app/views/welcome/index.html.erb

```diff
@@ -1,40 +1,110 @@
-<h1>RideBoard.app</h1>
+<%# Navbar%>
+<nav class="welcome">
+  <div class="nav-wrapper">
+    <a href="/" class="logo">RideBoard.app</a>
+    <a href="#" data-target="mobile-sidenav" class="sidenav-trigger"><i class="material-icons">menu</i></a>
+    <ul class="right hide-on-med-and-down">
+      <% if content_for? :rides_link %>
+        <li><%= yield :rides_link %></li>
+      <% end %>
```

Comment:
Delete this.  We aren't a layout, so no content is going to be injected

Reply:
Done.